test/plans/cutting.py

```diff
@@ -0,0 +1,42 @@
+import pycram.enums
+from pycram.process_module import simulated_robot
+from pycram.designators.action_designator import *
+from pycram.enums import Arms
+from pycram.designators.object_designator import *
+from pycram.designators.object_designator import BelieveObject
+
+
+def cutting_action_test():
+    """
+    This function is a test for the cutting action.
+    """
+    world = BulletWorld("DIRECT")
```

Comment:
tests should define the BulletWorld, not plans. Talk to Jonas for more information

Reply:
I already talked to @Tigul  and we decided that we want to have at least one test that runs the whole action. 
In the future, we will have some basic "if contact" events that we can check in those test cases.